README.md

```diff
@@ -5,6 +5,50 @@ BigQuery ETL
 
 Bigquery UDFs and SQL queries for building derived datasets.
 
+The `bqetl` CLI
+---
+
+The `bqetl` command-line tool aims to simplify working with the bigquery-etl repository by supporting
+common workflows, such as creating, validating and scheduling queries or adding new UDFs.
+
+`bqetl` can be installed by running `pip install mozilla-bigquery-etl` or by downloading the repository
+and running `pip install .`
```

Comment:
```suggestion
`bqetl` can be installed by running `pip install mozilla-bigquery-etl` or by cloning the repository
and running `pip install --editable .`
```

The `-e` or `--editable` flag ensures that the installed executable continues to reference the source code rather than making a static copy at the time of install.